src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,61 @@
+<div class="card-body container">
+  <div class="allcard row g-0">
+    <div class="card col-md-4 col-sm-12">
+      <p class="title">CONCEPT</p>
+      <figure class="img-wrapper">
+        <img src="../../../img/card-1.jpg" class="card-img" />
+      </figure>
+      <div class="card-triangle"></div>
+
+      <div class="card-body">
+        <p class="card-text">
+          「本屋」でも「自習室」でもない、新しい空間。それがこのカフェ＆コワーキングです。
+          <br />お好きなドリンクを飲みながらゆったりと気兼ねなく勉強や仕事に取り組んでいただけます。
+        </p>
+        <div class="module">
+          <div class="atoms-button">
+            <a href="#" class="url">コンセプト</a>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
そこで以下のように修正してみてください。
allcardクラスは特に適用するスタイルがなければ消して良いでしょう。g-0は子colにおける水平方向のpaddingを0にするものなので、今回は必要ありません。

```suggestion
  <div class="row row-cols-1 row-cols-md-3">  //row-cols-1 : 普段は1列で並べる, row-cols-md-3 : md以上で3列で並べる
  <div class="col">//rowクラスの下にgrid systemで並べたいブロックcolを設置する
    <div class="card">
      <p class="title">CONCEPT</p>
      <figure class="img-wrapper">
        <img src="../../../img/card-1.jpg" class="card-img" />
      </figure>
      <div class="card-triangle"></div>

      <div class="card-body">
        <p class="card-text">
          「本屋」でも「自習室」でもない、新しい空間。それがこのカフェ＆コワーキングです。
          <br />お好きなドリンクを飲みながらゆったりと気兼ねなく勉強や仕事に取り組んでいただけます。
        </p>
        <div class="module">
          <div class="atoms-button">
            <a href="#" class="url">コンセプト</a>
          </div>
        </div>
      </div>
    </div>
    </div>
```

Reply:
row-colsを使い子要素のcol達を並べています。colクラスにはgrid systemのcssによってデフォルトでpaddingが当たっているので、同じように残りのカードについても変更を行えば、カードの間に余白ができるはずです。

〈参考〉
・[Bootstrapのグリッドシステムについてまとめてみた](https://qiita.com/akatsuki174/items/53b7367b04ed0b066bbf)
・[Gutters](https://getbootstrap.jp/docs/5.0/layout/gutters/)
・[row-columns](https://getbootstrap.jp/docs/4.4/layout/grid/#row-columns)

Reply:
ありがとうございます。解決いたしました。